workbench-core/accounts/src/services/projectService.ts

```diff
@@ -6,42 +6,33 @@
 /* eslint-disable security/detect-object-injection */
 
 import { GetItemCommandOutput } from '@aws-sdk/client-dynamodb';
-import { AwsService, buildDynamoDBPkSk, resourceTypeToKey } from '@aws/workbench-core-base';
-import Boom from '@hapi/boom';
+import { AuthenticatedUser } from '@aws/workbench-core-authorization';
+import {
+  AwsService,
+  buildDynamoDBPkSk,
+  resourceTypeToKey,
+  uuidWithLowercasePrefix
+} from '@aws/workbench-core-base';
 
-interface Project {
-  pk: string;
-  sk: string;
-  id: string;
-  resourceType: string;
-  envMgmtRoleArn: string;
-  externalId: string;
-  accountHandlerRoleArn: string;
-  accountId: string;
-  awsAccountId: string;
-  createdAt: string;
-  createdBy: string;
-  dependency: string;
-  description: string;
-  encryptionKeyArn: string;
-  environmentInstanceFiles: string;
-  indexId: string;
-  name: string;
-  owner: string;
-  projectAdmins: string[];
-  subnetId: string;
-  updatedAt: string;
-  updatedBy: string;
-  vpcId: string;
-}
+import Boom from '@hapi/boom';
+import CostCenter from '../models/costCenter';
+import CreateProjectReqeust from '../models/createProjectRequest';
+import Project from '../models/project';
+import CostCenterService from './costCenterService';
 
 export default class ProjectService {
   private _aws: AwsService;
   private _resourceType: string = 'project';
+  private _tableName: string;
+  // TODO implement after dynamic AuthZ
+  // private _dynamicAuthorizationService: DynamicAuthorizationService;
 
   public constructor(constants: { TABLE_NAME: string }) {
     const { TABLE_NAME } = constants;
+    this._tableName = TABLE_NAME;
     this._aws = new AwsService({ region: process.env.AWS_REGION!, ddbTableName: TABLE_NAME });
+    // TODO implement after dynamic AuthZ
+    // this._dynamiAuthorizationService = new DynamicAuthorizationService();
```

Comment:
Amazing catch for commented out code. We would have caught during integration with dynamic authz work but I'll go ahead and change now since I am making other changes